### ISSUE TITLE
New 0.0.22 release breaks sdkman on Ubuntu 20.04

### ISSUE THREAD

**Issue by shakefu** [sdkman/sdkman-cli-native#110]
It looks like it was compiled against Glibc >= 2.32 but Ubuntu 20.04 only has 2.31...

... is this intentional or is there any way to continue to support Glibc 2.31?

```
    packer-ubuntu.amazon-ebs.ubuntu_base: [env-versions] /home/runners/.sdkman/libexec/version: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /home/runners/.sdkman/libexec/version) 
    packer-ubuntu.amazon-ebs.ubuntu_base: [env-versions] /home/runners/.sdkman/libexec/version: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by /home/runners/.sdkman/libexec/version) 
    packer-ubuntu.amazon-ebs.ubuntu_base: [env-versions] /home/runners/.sdkman/libexec/version: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /home/runners/.sdkman/libexec/version) 
```

**Comment by jbury** [comment]
+1!

Semi related, it seems like there is (or was) some issue with how releases actually update get.sdkman.io.  We were last able to successfully invoke sdkman native version 0.0.15 on Jan 26 (installing sdkman via the recommended `curl -s https://get.sdkman.io | bash` approach)

For some reason after Jan 28, the `get.sdkman.io` endpoint, jumped to getting version 0.0.22, and at this point our Ubuntu 20.04 base stopped being able to invoke sdkman post-installation.

**Comment by marc0der** [comment]
Thanks both, I'll see if I can set the build to explicitly use the earlier version of GLIBC.

**Comment by marc0der** [comment]
Hi @shakefu and @jbury,

I replicated this and have now tied our builds to GH runners using Ubuntu 20.04. I released native extensions 0.0.23, and tested this on Ubuntu 20.04 and 22.04, which seems to work well.

Could both of you please confirm that the latest release has resolved the problem?
